src/specification/models/position/index.json

```diff
@@ -0,0 +1,28 @@
+{
+  "additionalProperties": false,
+  "description": "A position represent a specific position",
```

Comment:
Maybe "A position is described by latitude and longitude" or something like that